src/main/java/StudentGroup.java

```diff
@@ -0,0 +1,57 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class StudentGroup {
+
+    private Student groupLeader;
+    private List<Student> students = new ArrayList<>();
+    private final List<String> tasks = new ArrayList<>();
+    Map<String, List<Student>> completedTasks = new HashMap<>();
+
+
+    public StudentGroup(Student groupLeader) {
+        this.groupLeader = groupLeader;
+    }
+
+    public Student changeGroupLeader(Student newGroupLeader) {
+        this.groupLeader = newGroupLeader;
+        return newGroupLeader;
+    }
+
+    public void addStudent(Student student) {
+        this.students.add(student);
+    }
+
+    public void removeStudent(Student student) {
+        this.students.remove(student);
+    }
+
+    public boolean addGroupTask(String task) {
+        tasks.add(task);
+        return false;
+    }
+
+    public void markTaskAsDone(Student student) {
```

Comment:
This method should take two arguments `Student student` and `String task`. Add code to find task in `this.tasks` and put or modify `completedTasks` map to mark task for student